unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentView.cs

```diff
@@ -13,42 +18,34 @@ public interface IExploreV2MenuComponentView
     /// </summary>
     GameObject go { get; }
```

Comment:
Exposing the GO can lead to bad practices and I'm not sure if it should part of the interface. Do we really need the GameObject to be part of the abstraction?

Reply:
You're right, it is not a good practice. Fixed in https://github.com/decentraland/unity-renderer/pull/1238/commits/f83c06799e4871ba364788203e838c0ea0872270. Thanks!